mlflow/tracking/client.py

```diff
@@ -444,6 +444,41 @@ def get_trace_info(self, request_id: str) -> TraceInfo:
         """
         return self._tracking_client.get_trace_info(request_id)
 
+    def search_traces(
+        self,
+        experiment_ids: List[str],
+        filter_string: Optional[str] = None,
+        max_results: int = SEARCH_TRACES_MAX_RESULTS,
+        order_by: Optional[List[str]] = None,
+        page_token: Optional[str] = None,
+    ):
```

Comment:
The client API should return `Trace`s  including `TraceData`, not just `TraceInfo`. For that reason, we should probably start with a default max results of ~ 100 traces.

Reply:
Yeah! :)